data/reservation.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+import jwt
+from sqlalchemy import Column, Integer, String, DateTime, Boolean
+from data.db import Base
+from server import app, bcrypt, BCRYPT_LOG_ROUNDS
+from sqlalchemy.sql.sqltypes import BOOLEAN, CHAR
+
+class Reservation(Base):
```

Comment:
Same as above comment.
Please use the existing file: https://github.com/ssw555tm012021Fall/halcyon-api/blob/main/data/reservation.py